scripts/triggers-end-to-end-tests/tests.ts

```diff
@@ -274,8 +274,8 @@ describe("storage emulator function triggers", () => {
     // default bucket
     expect(test.storageFinalizedTriggerCount).to.equal(1);
     expect(test.storageV2FinalizedTriggerCount).to.equal(1);
-    expect(test.storageMetadataTriggerCount).to.equal(0);
-    expect(test.storageV2MetadataTriggerCount).to.equal(0);
+    expect(test.storageMetadataTriggerCount).to.equal(1);
+    expect(test.storageV2MetadataTriggerCount).to.equal(1);
```

Comment:
Note for reviewer: Not sure how to run these locally/values seem to change between runs? Will investigate more on Monday but not sure if we even consider these tests before pushing?